packages/appframe/src/react/__stories__/index.story.js

```diff
@@ -0,0 +1,89 @@
+import { storiesOf } from '@storybook/react'
+import React, { useState } from 'react'
+
+import Button from '@pluralsight/ps-design-system-button'
+import { PageWidthLayout } from '@pluralsight/ps-design-system-layout'
+import * as Text from '@pluralsight/ps-design-system-text'
+
+import AppFrame from '../index.js'
+import { MockContent, SideNav, TopNav } from './shared.js'
+
+storiesOf('AppFrame', module)
+  .add('scrollable sidenav', _ => (
+    <AppFrame
+      topnav={<TopNav />}
+      sidenav={
+        <SideNav sections={[]}>
+          <div style={{ border: '1px dashed red', margin: 40, height: 2000 }} />
+        </SideNav>
+      }
+      sidenavOpen
+    >
+      <MockContent />
+    </AppFrame>
+  ))
+  .add('sidenav controlled', _ => {
+    function Story() {
+      const [open, setOpen] = useState(false)
+      const toggle = () => setOpen(!open)
+
+      return (
+        <AppFrame
+          onRequestSideNavClose={() => setOpen(false)}
+          onRequestSideNavOpen={() => setOpen(true)}
+          topnav={<TopNav onMobileMenuClick={toggle} />}
+          sidenav={({ visible }) => (
+            <SideNav collapsed={!visible} hideLabels={!visible} />
+          )}
+          sidenavOpen={open}
+        >
+          <PageWidthLayout>
+            <div
+              style={{
+                alignItems: 'center',
+                display: 'flex',
+                justifyContent: 'center',
+                minHeight: 'calc(100vh - 56px)'
+              }}
+            >
+              <Button onClick={toggle}>toggle sidenav</Button>
```

Comment:
For some reason, toggle close isn't working:

![toggleDoesntToggleClosed](https://user-images.githubusercontent.com/552044/89811832-76533600-dafc-11ea-9510-8f553b783f67.gif)

Reply:
i'll investigate